tests/series_only/is_between_test.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pytest
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+data = [1, 4, 2, 5]
+
+
+@pytest.mark.parametrize(
+    ("closed", "expected"),
+    [
+        ("left", [True, True, True, False]),
+        ("right", [False, True, True, True]),
+        ("both", [True, True, True, True]),
+        ("none", [False, True, True, False]),
+    ],
+)
+def test_is_between(constructor_series: Any, closed: str, expected: list[bool]) -> None:
+    ser = nw.from_native(constructor_series(data), series_only=True)
+    result = ser.is_between(1, 5, closed=closed)
+    compare_dicts({"a": result}, {"a": expected})
```

Comment:
Could you check what happens with different dtypes i.e. lower_bound and/or upper_bound being float instead of int?
I hope not, but we may need to do some casting?!

Reply:
Never mind it is merged 😂

Reply:
😄 still a good topic for a follow-up, I probably shouldn't merge so quickly some times

Reply:
I just checked with 1.0 and 5.0 and the test pass. Btw I forgot to remove series.is_between from backend completness, doing it now